lib/themes/dosomething/paraneue_dosomething/js/images/ImageUploadBeta.js

```diff
@@ -208,11 +208,16 @@ define(function(require) {
    */
   ImageUploadBeta.prototype.openModal = function() {
     this.enableSubmit();
+
     Modal.open(this.$cropModal,
       {
         animated: false,
       }
     );
+
+    // Turn off the click event that closes
+    // the modal when a user clicks outside of the modal.
+    this.$body.off("click", ".modal-container");
```

Comment:
This will turn off ability to close any modal by clicking the overlay until the page refreshes... this is a fine quick-fix, but we should have a more robust solution for the future. Can you add a `@TODO` to refactor?